fbgemm_gpu/include/fbgemm_gpu/fbgemm_cuda_utils.cuh

```diff
@@ -21,6 +21,7 @@
 // clang-format on
 
 #include <cuda.h>
+#include <cuda_bf16.h>
```

Comment:
@shintaro-iwasaki / @liligwu Do you have the suggestion on how to enable HIP support for cuda_bf16.h ? Looks like HIP tests are failing with the include of this header file: https://github.com/pytorch/FBGEMM/actions/runs/3654785378/jobs/6175505771

Reply:
@jianyuh , sorry to reply to you late. Please see the PR https://github.com/pytorch/FBGEMM/pull/1560